<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mesdialog</class>
 <widget class="QWidget" name="Mesdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_ques">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>600</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#008000;&quot;&gt;Loge xport succeeded!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_cancel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>300</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_ok">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>300</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
